checks/checks.py

```diff
@@ -344,17 +344,32 @@ def detect_file_type(pname):
     if not pname.is_file():
         return None
 
+    # Scripting files
     if pname.match('*.py'):
```

Comment:
Maybe using https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.suffix can be more efficient than using `match`?